Demisto-sdk release 1.31.3 (#4423)
demisto/demisto-sdk: CHANGELOG.md

# Changelog
## 1.31.3

> * Removed IN150 - invalid_siem_integration_name from both old and new validate flows. [#4420](https://github.com/demisto/demisto-sdk/pull/4420)

Reviewer: ```suggestion
* Removed IN150 due to invalid_siem_integration_name from both old and new validate flows. [#4420](https://github.com/demisto/demisto-sdk/pull/4420)
```

> * Removed RM108 & RM112 from the old validate. [#4408](https://github.com/demisto/demisto-sdk/pull/4408)

Reviewer: ```suggestion
* Removed RM108 and RM112 from the old validate format. [#4408](https://github.com/demisto/demisto-sdk/pull/4408)
```

* Added the **-tpb** option to the **upload** command. When provided, all the test playbooks for the given pack will be uploaded. [#4402](https://github.com/demisto/demisto-sdk/pull/4402)

> * Refactored validation GR100, that checks whether a content item attempts to use other content items which are not supported in all of the marketplaces of the content item. [#4282](https://github.com/demisto/demisto-sdk/pull/4282)

Reviewer: ```suggestion
* Modified GR100, that checks whether a content item attempts to use other content items which are not supported in all of the marketplaces of the content item. [#4282](https://github.com/demisto/demisto-sdk/pull/4282)
```

* Added the ability to use git.Repo object to initialize GitUtil. [#4387](https://github.com/demisto/demisto-sdk/pull/4387)

> * Print timestamp in console in every environment. [#4412](https://github.com/demisto/demisto-sdk/pull/4412)

Reviewer: ```suggestion
* Added timestamp print in console. [#4412](https://github.com/demisto/demisto-sdk/pull/4412)
```

* Add a pre-commit hook to validate file permission/mode changes in Packs directory. [#4389](https://github.com/demisto/demisto-sdk/pull/4389)

> * Fixed an issue where the Relative path regex wasn't written correctly. This fix affect RM108, RM114 validation in the new validate. [#4413](https://github.com/demisto/demisto-sdk/pull/4413)

Reviewer: ```suggestion
* Fixed an issue where the relative path regex wasn't written correctly. This fix affect RM108 and RM114 in the new validate. [#4413](https://github.com/demisto/demisto-sdk/pull/4413)
```

* Fixed an issue where new validate failed when attempting to parse renamed files. [#4411](https://github.com/demisto/demisto-sdk/pull/4411)
* Fixed an issue where playbook saved due to a failed attempt to dump task object. [#4416](https://github.com/demisto/demisto-sdk/pull/4416)

> * Added support to the (internal) **test-content** command to work properly with multiple cloud servers. [#4321](https://github.com/demisto/demisto-sdk/pull/4321)

Reviewer: ```suggestion
* Added support for the (internal) ***test-content*** command to work properly with multiple cloud servers. [#4321](https://github.com/demisto/demisto-sdk/pull/4321)
```
